pulp_2to3_migration/app/pre_migration.py

```diff
@@ -116,6 +118,14 @@
                 pulp2detail_pb.total -= 1
                 pulp2detail_pb.save()
                 continue
+        elif record._content_type_id in mutable_content_models.keys():
```

Comment:
Since we process content a type by type, does it make sense  to check it before the for loop and have 2 loops, one for immutable content and one for mutable one? This way we check content type only once and not on every iteration.

Reply:
yes i can do that, however booth of the loops would repeat the code where we check for the `if record._last_updated == last_updated`